bindgen-tests/tests/headers/macro_func_wrapper.h

```diff
@@ -0,0 +1,4 @@
+void func(int, int);
+
+#define Y 7
+#define wrapper_func(x) func(x, Y)
```

Comment:
Seems like it is currently ignored, probably since macro parsing stops if the macro name is encountered.

Reply:
First, the macro cannot be generated since it thinks it is a recursive definition:

```
Cannot generate function macro: RecursiveDefinition("func")
```

Next, the `func` function is not generated since the macro previously called `saw_function`.

So apart from the recursive definition issue, we need to differentiate between `saw_function` and `saw_fn_macro`.

Reply:
This edge case is now supported.